client/landing/gutenboarding/components/plans/plans-table/plan-item.tsx

```diff
@@ -67,7 +69,9 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 					<div className="plan-item__price-amount" data-is-loading={ ! price }>
 						{ price || nbsp }
 					</div>
-					<div className="plan-item__price-note">{ __( 'per month, billed yearly' ) }</div>
+					<div className="plan-item__price-note">
+						{ isFree ? __( 'free forever' ) : __( 'per month, billed yearly' ) }
```

Comment:
Alternatively, we could check for the price to be `0` (regardless of currency)

Reply:
The price has a currency symbol that might come before or after it.